packages/Webkul/Velocity/src/Resources/assets/js/UI/components/sidebar.vue

```diff
@@ -5,7 +5,7 @@
         :class="`sidebar ${addClass ? addClass : ''}`"
         v-if="slicedCategories && slicedCategories.length > 0"
     >
-        <ul type="none" style="margin-bottom: 0">
+        <ul type="none" class="main-cat">
```

Comment:
Give some meaningful class names. Don't use the short version.

Reply:
Done